tests/cluster/tests/29-slot-migration-response.tcl

```diff
@@ -1,50 +0,0 @@
-# Tests for the response of slot migrations.
```

Comment:
Why did you delete this test file?

Reply:
Its test is included in the new file.

Reply:
Oh, I see now. At a glance it didn't make sense, but looking more closely you are correct.